test/externalTests/README.md

```diff
@@ -0,0 +1,66 @@
+## Solidity external tests
+This directory contains scripts for compiling some of the popular open-source projects using the
+current version of the compiler and running their test suites.
+
+Since projects often don't use the latest compiler, we keep a fork of each of these projects
+at https://github.com/solidity-external-tests/. If changes are needed to make a project work with the
+latest version of the compiler, they're maintained as a branch on top if the upstream master branch.
+This is especially important for testing our `breaking` branch because we can not realistically expect
+external projects to be instantly compatible with a compiler version that has not been released yet.
+It also isolates us from upstream changes to some degree - their changes won't affect our test suite
+until we explicitly pull them.
+
+### Recommended workflow
+
+#### Adding a new external project
+1. Create a fork of the upstream repository in https://github.com/solidity-external-tests/. If the
+    project consists of several repositories, fork them all.
+2. Remove all the branches except for main one (`master`, `develop`, `main`, etc). This branch is
+    going to be always kept up to date with the upstream repository and should not contain any extra
+    commits.
+3. Create two new versions of the main branch, representing versions of the compiler currently in
+    `develop` and `breaking`. E.g. if the latest Solidity version is 0.7.5 and the main branch of the
+    external project is called `master`, create `master_070` and `master_080`. This is where we will
+    be adding our own commits. The one corresponding to the newer Solidity version should always be
+    on top on the older one. E.g. if a change is needed to keep the project
+    compilable using Solidity 0.7.x, add it in `master_070` and rebase `master_080` on top of it.
+    If it is only needed for the compiler from its `breaking` branch, add it only in `master_080`.
+    - The fewer commits in these branches, the better. Ideally, any changes needed to make the compiler
+        work should be submitted upstream and the branches should just be tracking the upstream
+        one without any extra commits.
```

Comment:
```suggestion
        ones without any extra commits.
```

Reply:
or ``the respective upstream one``. Although I'm starting to nitpick :-).

Reply:
I meant that there's one upstream branch (e.g. `master`) and multiple branches are tracking it (`master_070`, `master_080`).

We do have multiple upstream branches in some cases (`develop` and `v2` for Gnosis) but I didn't want to complicate the description by adding that. It should be pretty obvious how it works with multiple upstreams when you get how it does with one.

Reply:
Ok... I'd just drop the entire thing after the ``and`` then :-). But as you like :-)!

Reply:
I rewrote it like this:
```
Ideally, any changes needed to make the compiler work should be submitted upstream and our branches should just be tracking the main upstream branch without any extra commits.              
```